test/tstDistributedTree.cpp

```diff
@@ -123,7 +123,7 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(hello_world, DeviceType, ARBORX_DEVICE_TYPES)
       "Testing::nearest_queries", 1);
   auto nearest_queries_host = Kokkos::create_mirror_view(nearest_queries);
   nearest_queries_host(0) = ArborX::nearest<ArborX::Point>(
-      {{0.0 + comm_size - 1 - comm_rank, 0., 0.}},
+      {{0.f + comm_size - 1 - comm_rank, 0., 0.}},
```

Comment:
Do we really need that cast?
(not blocking just trying to understand)

Reply:
Yes.